src/main/java/io/github/raeperd/realworld/domain/article/Category.java

```diff
@@ -0,0 +1,24 @@
+package io.github.raeperd.realworld.domain.article;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "categories")
+public class Category {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "created_at", nullable = false)
+    private LocalDateTime createdAt;
```

Comment:
Consider adding a unique constraint to the 'name' column in the Category entity to ensure that category names are unique. This will help maintain data integrity and prevent duplicate categories.